modular_skyrat/modules/verbs/code/subtle.dm

```diff
@@ -24,25 +27,13 @@
 	else if(!params)
 		subtle_emote = stripped_multiline_input(user, "Choose an emote to display.", "Subtle", null, MAX_MESSAGE_LEN)
 		if(subtle_emote && !check_invalid(user, subtle_emote))
-			var/type = input("Is this a visible or hearable emote?") as null|anything in list("Visible", "Hearable")
-			switch(type)
-				if("Visible")
-					emote_type = EMOTE_VISIBLE
-				if("Hearable")
-					emote_type = EMOTE_AUDIBLE
-				else
-					alert("Unable to use this emote, must be either hearable or visible.")
-					return
 			subtle_message = subtle_emote
 		else
 			return FALSE
 	else
 		subtle_message = params
 		if(type_override)
 			emote_type = type_override
-	. = TRUE
-	if(!can_run_emote(user))
-		return FALSE
```

Comment:
That check still needs to be done before sending the emote, otherwise you could probably be dead and send a subtler. I know it sounds stupid, but it's still necessary.

Reply:
Ack-